hazelcast/src/main/java/com/hazelcast/instance/impl/Node.java

```diff
@@ -911,4 +925,101 @@ public boolean isMaster(Address address) {
         Address masterAddress = clusterService.getMasterAddress();
         return server.doAddressesMatch(masterAddress, address);
     }
+
+    public void printSecurityInfo() {
+        securityLogger.info(String.format(
+                "Enable DEBUG/FINE log level for log category %1$s "
+                        + " or use -D%1$s system property to see 🔒 security recommendations and the status of current config.",
```

Comment:
I don't think that -Dcom.hazelcast.system.security is the right name for the system property, is it? I I guess that's a overlook?

Reply:
I made the name intentionally the same as the log category. But if it's confusing, I could change it.

Reply:
Do we have any other system property that doesn't start with `hazelcast.`?